packages/apollos-data-connector-postgres/src/media/migrations/004_add_metadata_column.js

```diff
@@ -0,0 +1,17 @@
+import { DataTypes } from 'sequelize';
+
+async function up({ context: queryInterface }) {
+  await queryInterface.addColumn('media', 'metadata', {
+    type: DataTypes.JSONB,
+    allowNull: false,
+    defaultValue: {},
+  });
+}
+
+async function down({ context: queryInterface }) {
+  await queryInterface.removeColumn('media', 'metadata');
+}
+
+const name = '004-add_metadata_column';
+
+module.exports = { up, down, name, order: 6 };
```

Comment:
`AddIdToMedia003` has order `6` :) 
```suggestion
module.exports = { up, down, name, order: 7 };
```

Reply:
@vinnyjth ahh got it. Thanks!